en/docs/references/extend/user-mgt/self-register-verification.md

```diff
@@ -0,0 +1,186 @@
+# Verification methods of Self-Registration
+
+WSO2 Identity Server (WSO2 IS) supports self-registration, allowing users to set up their accounts and receive a confirmation when the account is created.
+
+WSO2 Identity Server allows users to confirm their accounts via a user-preferred communication channel. Based on the user preference, the user self-registration process supports email or mobile as the preferred account confirmation channel for the user.
```

Comment:
email or mobile? 
in the next line we say SMS and EMAIL?

@divyaamunugama is this right?

Reply:
better stick to one term `mobile` or `SMS`

Reply:
It has to be SMS and EMAIL.
When it comes to mobile there can be multiple ways.
- SMS
- Call and ask the call to answer etc